index.src.html

```diff
@@ -572,12 +583,25 @@ <h4 id="clear-cache">
     2.  Let |cache list| be the set of entries from the <a>network cache</a> whose `target URI`
         [=url/host=] is identical to |host|.
 
-    3.  For each |entry| in |cache list|:
+        1.  For each |entry| in |cache list|:
    
-        1.  Remove |entry| from the <a>network cache</a>.
+            1.  Remove |entry| from the <a>network cache</a>.
+
+        2.  If a user agent implements caches beyond a pure <a>network cache</a>, it MUST remove
+            all entries from those caches which match |origin|.
+
+    3.  For each |traversable| in the user agent's set of [=top-level traversable=]:
+
+        1.  For each |entry| in the |traversable|'s [=session history entries=]:
+
+            1.  Let |state| be |entry|'s <a>document state</a> whose `origin` attribute is identical
+                to |host|.
```

Comment:
This sentence doesn't make sense to me.

I think you want to look at |entry|'s [=session history entry/document state=]'s [=document state/origin=].

Then, you need to compare it to |origin| (not |host|, I don't think??). But you can't just say identical to; you need to use [=same origin=].

And then the idea is you want to [=continue=] the loop if there's no match. You can't just assume there's always a match.

Reply:
```
2.  Let |cache list| be the set of entries from the <a>network cache</a> whose `target URI`
        [=url/host=] is identical to |host|.
```
I was copying the terms from line 583 above, which is not accurate for this BFCache clearing case. I have updated the content following the suggestions.

Do we need to explicitly [=continue=] the iteration in the spec if we don't have any steps to skip over below?

Reply:
No need to do so.